crates/history/tests/query.rs

```diff
@@ -0,0 +1,52 @@
+#![cfg(feature = "query")]
```

Comment:
I would also run these tests inside a browser:
```rust
#[cfg(target_arch = "wasm32")]
use wasm_bindgen_test::{wasm_bindgen_test as test, wasm_bindgen_test_configure};
#[cfg(target_arch = "wasm32")]
wasm_bindgen_test_configure!(run_in_browser);
```

Reply:
Neat! I was able to run the headless browser tests locally with this.